pkgs/servers/monitoring/prometheus/pve-exporter.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, python3Packages
+}:
+
+with python3Packages;
+
+buildPythonApplication rec {
```

```suggestion
python3.pkgs.buildPythonApplication rec {
```

I'd recommend against using `with python3Packages` in such a coarse scope.